app/src/pages/project/SpansTable.tsx

```diff
@@ -230,6 +230,15 @@ export function SpansTable(props: SpansTableProps) {
     {
       id: "select",
       maxSize: 10,
+      header: ({ table }) => (
+        <IndeterminateCheckboxCell
+          {...{
+            checked: table.getIsAllRowsSelected(),
+            indeterminate: table.getIsSomeRowsSelected(),
+            onChange: table.getToggleAllRowsSelectedHandler(),
+          }}
+        />
+      ),
```

Comment:
Small thing. It might be worth hiding the header checkbox when the table is empty.

Reply:
Eh, punt ;) It's not checkable anyways